### DISCUSSION TITLE
Changing the shape of bounding boxes

### DISCUSSION THREAD

**Discussion by Bwwwah** [ultralytics/yolov5#7208] [Q&A]
Hello ! I would like to know if anyone knows how to change the shape of the bounding boxes please ? I can't find in 'detect.py'.

**Comment by Bwwwah** [comment]
If anyone also knows how to change the color that would be cool.

**Reply by glenn-jocher** [reply]
@Bwwwah 👋 Hello! Thanks for asking about **handling inference results**. YOLOv5 🚀 [PyTorch Hub](https://github.com/ultralytics/yolov5/issues/36) models allow for simple model loading and inference in a pure python environment without using `detect.py`. 

### Simple Inference Example

This example loads a pretrained YOLOv5s model from PyTorch Hub as `model` and passes an image for inference. `'yolov5s'` is the YOLOv5 'small' model. For details on all available models please see the [README](https://github.com/ultralytics/yolov5#pretrained-checkpoints). Custom models can also be loaded, including custom trained PyTorch models and their [exported](https://github.com/ultralytics/yolov5/issues/251) variants, i.e. ONNX, TensorRT, TensorFlow, OpenVINO YOLOv5 models.

```python
import torch

# Model
model = torch.hub.load('ultralytics/yolov5', 'yolov5s')  # or yolov5m, yolov5l, yolov5x, etc.
# model = torch.hub.load('ultralytics/yolov5', 'custom', 'path/to/best.pt')  # custom trained model

# Images
im = 'https://ultralytics.com/images/zidane.jpg'  # or file, Path, URL, PIL, OpenCV, numpy, list

# Inference
results = model(im)

# Results
results.print()  # or .show(), .save(), .crop(), .pandas(), etc.

results.xyxy[0]  # im predictions (tensor)
results.pandas().xyxy[0]  # im predictions (pandas)
#      xmin    ymin    xmax   ymax  confidence  class    name
# 0  749.50   43.50  1148.0  704.5    0.874023      0  person
# 2  114.75  195.75  1095.0  708.0    0.624512      0  person
# 3  986.00  304.00  1028.0  420.0    0.286865     27     tie
```
<img width="550" src="https://user-images.githubusercontent.com/26833433/149679662-b2021c78-59b2-4f52-b768-1e5161537c85.png">

See [YOLOv5 PyTorch Hub Tutorial](https://github.com/ultralytics/yolov5/issues/36) for details.

Good luck 🍀 and let us know if you have any other questions!

**Reply by Bwwwah** [reply]
Thank you !
